src/lib/github/commitStatus.ts

```diff
@@ -20,11 +20,20 @@ export const sendCommitStatus = async (snykDeltaBinaryResult: number, snykProjec
       data.target_url = `https://app.snyk.io/org/${snykProjectDetails.orgName}/project/${snykProjectDetails.projectID}`
     }
 
+    // Overwrite snykDeltaBinaryResult because setPassIfNoBaselineFlag is true
+    if (issueFoundNoBaseline === true)
+    {
+      snykDeltaBinaryResult = 0;
```

Comment:
this is not necessary. snyk-delta returns 0 if the option to passifnobaseline has been provide. issueFoundNoBaseline should only be used to tweak the description